roles/s2i/cakephp_backup_restore/tasks/main.yml

```diff
@@ -40,7 +38,7 @@
     )}}'
 
 # Simulate DR and restore
-- name: Delete cake php app
+- name: Delete cakephp app
   k8s:
     state: absent
     definition:
```

Comment:
This doesn't have to happen in this PR, but could we also add checks for the buildconfigs/imagestreams that are being created? These are resources we want to ensure are being restored and what makes this sample app interesting. All of the examples seem to just be checking that the deployment comes up.

Reply:
Sure, I will add this in new PR